sync/sync-impl/src/main/res/layout/fragment_recovery_code.xml

```diff
@@ -0,0 +1,212 @@
+                android:layout_marginStart="@dimen/keyline_5"
+                android:layout_marginEnd="@dimen/keyline_5"
+                app:layout_constraintTop_toBottomOf="@id/content_illustration"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:gravity="center"
+                app:textType="secondary"
+                app:typography="h1"
+                android:text="@string/sync_recovery_code_title"/>
+
+            <com.duckduckgo.mobile.android.ui.view.text.DaxTextView
+                android:id="@+id/content_body"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/keyline_5"
+                app:layout_constraintTop_toBottomOf="@id/content_title"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:gravity="center"
+                app:textType="primary"
+                app:typography="body1"
+                android:text="@string/sync_recovery_code_content"/>
+
+            <com.facebook.shimmer.ShimmerFrameLayout
+                android:id="@+id/recoveryCodeSkeleton"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:layout_margin="20dp"
+                app:layout_constraintTop_toBottomOf="@id/content_body"
+                android:padding="@dimen/keyline_5">
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+                    <com.duckduckgo.mobile.android.ui.view.SkeletonView
+                        android:layout_width="96dp"
+                        android:layout_height="96dp"/>
+                    <com.duckduckgo.mobile.android.ui.view.SkeletonView
+                        android:layout_width="match_parent"
+                        android:layout_marginStart="@dimen/keyline_4"
+                        android:layout_height="96dp"/>
+
+                </LinearLayout>
+
+            </com.facebook.shimmer.ShimmerFrameLayout>
+
+            <LinearLayout
+                android:id="@+id/recover_code_container"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintStart_toStartOf="parent"
+                android:visibility="invisible"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:layout_margin="20dp"
+                app:layout_constraintTop_toBottomOf="@id/content_body"
+                android:background="@color/black3"
```

Comment:
@malmstein lint fails here because we are using this color. How can I give this viewgroup an specific background color? 

As a reference: this is the UI
![image](https://user-images.githubusercontent.com/4747865/227526222-ce9e2d52-ed8b-479c-a735-969c89568c9a.png)